packages/rpc-subscriptions-api/src/logs-notifications.ts

```diff
@@ -4,33 +4,59 @@ import type { Commitment, SolanaRpcResponse, TransactionError } from '@solana/rp
 
 type LogsNotificationsApiNotification = SolanaRpcResponse<
     Readonly<{
-        // Error if transaction failed, null if transaction succeeded
+        /** Error if transaction failed, null if transaction succeeded. */
         err: TransactionError | null;
-        // Array of log messages the transaction instructions output during execution,
-        // null if simulation failed before the transaction was able to execute
-        // (for example due to an invalid blockhash or signature verification failure)
-        logs: readonly string[] | null;
```

Comment:
Quite sure this was wrong.

https://github.com/solana-foundation/solana-com/pull/387